grakn-engine/src/main/java/ai/grakn/engine/GraknEngineServer.java

```diff
@@ -203,12 +203,17 @@ private static void checkAuthorization(Service spark, Request request) {
             boolean authenticated;
             try {
                 if (request.headers("Authorization") == null || !request.headers("Authorization").startsWith("Bearer ")) {
-                    throw new GraknEngineServerException(400, "Authorization field in header corrupted or absent.");
+                    throw new GraknEngineServerException(401, "Authorization field in header corrupted or absent.");
```

Comment:
I'm not sure...if it's not a core error message, it shouldn't be in core. But it's a good idea them all in  one place, preferably in a ResourceBundle actually for i18n.